fs/xfstests.py

```diff
@@ -106,6 +101,16 @@ def setUp(self):
         self.disk_mnt = self.params.get('disk_mnt', default='/mnt/loop_device')
         self.dev_type = self.params.get('type', default='loop')
         self.fs_to_test = self.params.get('fs', default='ext4')
+        if self.dev_type == 'nvdimm':
+            packages.extend(['ndctl', 'parted'])
+            if self.detected_distro.name == 'rhel':
+                packages.extend(['daxctl'])
+
+        for package in packages:
+            if not sm.check_installed(package) and not sm.install(package):
```

Comment:
club this to above `https://github.com/avocado-framework-tests/avocado-misc-tests/blob/af2fbb8363bcd6d4b9709c69adf77f77ab7ddcd1/fs/xfstests.py#L59` as based on nvdim is dev type extend package list there it self

Reply:
Sure